pkgs/applications/editors/neovim/utils.nix

```diff
@@ -0,0 +1,178 @@
+{ lib
+, vimUtils
+, nodejs
+, neovim-unwrapped
+, bundlerEnv
+, ruby
+, pythonPackages
+, python3Packages
+, writeText
+, wrapNeovim2
+}:
+let
+  # vimWithRC pass it args
+  # need a function that generates wrapperArgs + customRC and then one can create its own package from that.
+  # should let the user write its own file.
```

Comment:
Could you expand this comment a bit? I don't quite understand it.

Reply:
that's normal that doesn't mean a thing my bad I 've prepared a much better test sry xD